packages/synapse-react-client/src/components/Forum/ForumTable.tsx

```diff
@@ -130,7 +130,7 @@ export const ForumTable: React.FC<ForumTableProps> = ({
 
   const threads = useMemo(
     () => data?.pages.flatMap(page => page.results) ?? [],
-    [data],
+    [data?.pages],
```

Comment:
Properly update the `threads` variable when a page of infinite data is added/removed (when data is added, the reference to `data` does not update when `data.pages` does)